File: UltimateFestivalOrganizer/UltimateFestivalOrganizer.DAL.Common/Domain/Artist.cs
using DAL.Common.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UltimateFestivalOrganizer.DAL.Common.Attributes;

namespace UltimateFestivalOrganizer.DAL.Common.Domain
{
    [Table("Artist")]
    public class Artist : BaseEntity
    {
        public string Email { get; set; }
        public string Name { get; set; }
        public string Link { get; set; }
        public string Picture { get; set; }
        public Catagory Catagory { get; set; }
        public string Country { get; set; }

        public bool Deleted { get; set; }

    }
}

File: UltimateFestivalOrganizer/UltimateFestivalOrganizer.Web/src/main/java/at/wlumetsberger/ufo/models/Artist.java
package at.wlumetsberger.ufo.models;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;

public class Artist extends BaseEntity {
	private static final long serialVersionUID = 4950996861427703886L;
	@Getter @Setter
	@SerializedName("Email")
	private String email;
	@Getter @Setter
	@SerializedName("Name")
	private String name;
	@Getter @Setter
	@SerializedName("Link")
	private String link;
	@Getter @Setter
	@SerializedName("Picture")
	private String picture;
	@Getter @Setter
	@SerializedName("Catagory")
	private Catagory catagory;
	@Getter @Setter
	@SerializedName("Country")
	private String country;
}
